homeworks/react-continued-3/yana_yahello/src/lectures.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
file with react component better to name in `PascalCase`

Reply:
renamed